collections/_article/release-candidate-godot-4-0-2-rc-1.md

```diff
@@ -0,0 +1,147 @@
+---
+title: "Release candidate: Godot 4.0.2 RC 1"
+excerpt: "With the GDC 2023 past us, we are getting ready to officially being the 4.1 dev cycle. But that doesn't mean current Godot 4.0 users have to wait long, because here's a release candidate for 4.0.2, with a handful of fixes and extra enhancements!"
```

Comment:
```suggestion
excerpt: "With the GDC 2023 past us, we are getting ready for the 4.1 dev cycle. But that doesn't mean current Godot 4.0 users have to wait, because here's a release candidate for 4.0.2, with a handful of fixes and extra enhancements!"
```

Simplification :)